types/src/content/index.ts

```diff
@@ -75,8 +76,8 @@ export class EnglishAuctionParams extends JoyStructDecorated({
   starting_price: Balance,
   buy_now_price: Option.with(Balance),
   whitelist: BTreeSet.with(MemberId),
-  end: BlockNumber,
-  auction_duration: BlockNumber,
+  duration: BlockNumber,
+  starts_at: Option.with(BlockNumber),
```

Comment:
Order needs to match the definition in runtime.
```suggestion
  starts_at: Option.with(BlockNumber),
  duration: BlockNumber,
```